src/etc/inc/plugins.inc.d/dpinger.inc

```diff
@@ -69,7 +69,7 @@ function dpinger_services()
 function dpinger_configure()
 {
     return [
-        'monitor' => ['dpinger_configure_do:3'],
+        'monitor' => ['dpinger_configure_do:2'],
```

Comment:
nice catch, remnant from the $bootup flag :)

Reply:
:)